taplo.hcl

```
@@ -0,0 +1,24 @@
+homepage = "https://github.com/tamasfe/taplo"
+description = "A TOML toolkit written in Rust"
+binaries = ["taplo"]
+test = "taplo --version"
+
+darwin {
+  source = "https://github.com/tamasfe/taplo/releases/download/${version}/taplo-${os}-${xarch}.gz"
+    on unpack {
+      rename { from = "${root}/taplo-${os}-${xarch}" to = "${root}/taplo" }
+    }
+}
+
+linux {
+  source = "https://github.com/tamasfe/taplo/releases/download/${version}/taplo-${os}-${xarch}.gz"
+  on unpack {
+    rename { from = "${root}/taplo-${os}-${xarch}" to = "${root}/taplo" }
+  }
+}
```

Wait a second. Aren't these identical???
```suggestion
source = "https://github.com/tamasfe/taplo/releases/download/${version}/taplo-${os}-${xarch}.gz"

on unpack {
  rename { from = "${root}/taplo-${os}-${xarch}" to = "${root}/taplo" }
}
```